richtext/plugins/org.polarsys.kitalpha.richtext.nebula.widget/src/org/polarsys/kitalpha/richtext/nebula/widget/MDENebulaRichTextConfiguration.java

```diff
@@ -122,5 +125,19 @@ public void setToolbarPosition(String postion) {
 
 		setOption(MDERichTextConstants.TOOLBAR_POSITION, postion);
 	}
-
+	
+	@Override
+	public void customizeToolbar() {
+		if (Platform.OS_LINUX.equals(Platform.getOS()) && !Boolean.getBoolean("org.polarsys.kitalpha.richtext.webkit.gtk.compatibility.workaround.disable")) {
```

Comment:
Hi @zgrtls, 

Could you give me a pointer to the constant utility class that I can use ? 
or the location where I can introduce it (and pointer to similar class in Capella)  ?

Reply:
I don't see any nice place to put it. It could got in an util namespace of a kitalpha.commun plugin except that only the ui exists today. Let's leave it like that for now.